src/map.h

```diff
@@ -29,6 +29,7 @@
 #include "cata_utility.h"
 #include "faction.h"
 #include "point.h"
+#include "scent_block.h"
```

Comment:
If you only need this for `scent_block` in `spread_gas`, a forward declaration should be enough.